src/App.css

```diff
@@ -12,5 +12,5 @@
 }
 
 body {
```

Comment:
- Nice Job so far. Please Use the information detailed in Zeplin to style the website. ([Link to Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378))
- Kindly make sure your final design should match [Zeplin's design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) Because it is the requirement of the project.
- Your Design is below:
![Capture](https://user-images.githubusercontent.com/63972670/186252954-fc89ce46-e6c0-4018-9f74-47b94debba5a.PNG)
- 
- Zeplin design:
![1](https://user-images.githubusercontent.com/63972670/186253206-657ebffd-daa9-4c9b-ba94-02b8887f4c33.PNG)

Reply:
resolved